curve25519-dalek/src/backend/serial/curve_models/mod.rs

```diff
@@ -383,13 +385,25 @@ impl ProjectivePoint {
         let XX = self.X.square();
         let YY = self.Y.square();
         let ZZ2 = self.Z.square2();
-        let X_plus_Y = &self.X + &self.Y;
-        let X_plus_Y_sq = X_plus_Y.square();
         let YY_plus_XX = &YY + &XX;
         let YY_minus_XX = &YY - &XX;
 
+        cfg_if! {
```

Comment:
Using the fully-qualified name of the macro avoids needing to add a `use` statement above, which is _slightly_ nicer is reduces the number of opportunities to introduce a merge conflict in the future. A merge conflict in the import declarations is quite likely since the change often.
```suggestion
        cfg_if::cfg_if! {
```